flow/display_list_utils.h

```diff
@@ -190,20 +190,32 @@ class SkMatrixDispatchHelper : public virtual Dispatcher,
 class ClipBoundsDispatchHelper : public virtual Dispatcher,
                                  private virtual SkMatrixSource {
  public:
-  void clipRect(const SkRect& rect, bool isAA, SkClipOp clip_op) override;
-  void clipRRect(const SkRRect& rrect, bool isAA, SkClipOp clip_op) override;
-  void clipPath(const SkPath& path, bool isAA, SkClipOp clip_op) override;
+  ClipBoundsDispatchHelper() : ClipBoundsDispatchHelper(nullptr) {}
+
+  ClipBoundsDispatchHelper(const SkRect* cull_rect)
+      : has_clip_(cull_rect),
+        bounds_(cull_rect && !cull_rect->isEmpty() ? *cull_rect
+                                                   : SkRect::MakeEmpty()) {}
+
+  void clipRect(const SkRect& rect, bool is_aa, SkClipOp clip_op) override;
+  void clipRRect(const SkRRect& rrect, bool is_aa, SkClipOp clip_op) override;
+  void clipPath(const SkPath& path, bool is_aa, SkClipOp clip_op) override;
```

Comment:
I think the style guide would want these to be capitalized, but it looks like other things in this file are not, and we're not super consistent in the repo. 

That said, it'd probably be good for new code to prefer using PascalCase for method names in C++. But if you'd rather make that consistent in a separate PR that SGTM too.

Reply:
This is not new code in this PR. If we should change these, that should be done outside of this PR.

Reply:
In revisiting this in https://github.com/flutter/engine/pull/28753 I decided to leave these specific method names consistent with the names in SkCanvas and SkPaint that they are replacing, but I looked to make method names consistent with the style guidelines. Please follow up in that PR with further discussion, but I wanted to make a note here to remind myself of where further discussion of these suggestions went.